src/codegen.cpp

```diff
@@ -6432,6 +6432,7 @@ extern "C" void jl_init_codegen(void)
 #endif
 
 #ifdef USE_POLLY
+    cl::ParseEnvironmentOptions("Julia", "JULIA_POLLY_ARGS");
```

Comment:
Yes, I suppose so..
It is somewhat strange, that I can't find a mechanism in llvm::cl for scoping the application of arguments. Nor can one restrict the parsing of opts to an optioncategory or something like that.
I will look further into that.